src/main/java/uk/gov/hmcts/dts/fact/mapit/MapitClient.java

```diff
@@ -12,4 +14,9 @@
 
     @GetMapping("${mapit.endpoint.postcode-search}/partial/{postcode}")
     MapitData getMapitDataWithPartial(@PathVariable("postcode") String postcode);
+
+    // This can be narrowed to specific types using 'type' query parameter in future e.g. ?types=MTD,MTW
+    // https://mapit.mysociety.org/docs/#api-multiple_areas
+    @GetMapping("${mapit.endpoint.area-search}/{area}")
```

Comment:
@KianKwa I've queried the types required on the main story. I think allowing all types for might be ok but I'd appreciate a second-opinion on this.

Reply:
@KianKwa See comments on main story. I queried all the existing local authorities in FaCT with mapit and found that they all belong to one of 4 'types'. I've therefore updated the url to narrow down the check to these types only.